ONSdigital/dp-frontend-release-calendar
# Implement call to API

### What

Call the release calendar API to read legacy releases

### How to review

Check code and unit tests.

Optionally run it:
- Checkout branch `feature/legacy-endpoint` of  `dp-release-calendar-api` and run it: `make debug`
- Run `zebedee` in reader mode: `./run-reader.sh`
- Run latest version of `dp-api-router`: `make debug`
- Tun this service: `make debug`
- Find an existing release page in your local zebedee content and navigate to it, for example: http://localhost:27700/releases/adoption
- Check some content is generated (no styling)

### Who can review
Anyone but me
